### PR TITLE
Upgrade the spring-core dependency

### PR DESCRIPTION
**Pull request by MarcelBochtler**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -55,8 +55,8 @@ postgresEmbeddedVersion = 0.13.3
 reflectionsVersion = 0.9.12
 retrofitVersion = 2.9.0
 semverVersion = 3.1.0
-simpleExcelVersion = 1.1
-springCoreVersion = 5.2.8.RELEASE
+simpleExcelVersion = 1.2
```
Comment (sschuberth) [inline gradle.properties:58]: I believe there was a reason for us not to upgrade this. IIRC, it was because the [changelog](https://github.com/tobyweston/simple-excel/releases/tag/simple-excel-1.2) documents the upgrade to Apache POI 4.1.0, which would break the Excel export for us, whose test is currently disabled because of [this](https://github.com/tobyweston/simple-excel/issues/14). Yeah, the Excel reporter stuff is a bit messy, but I hope to get rid of it completely soon.

TL;DR please drop this for now.

Anyway, please upgrade only one dependency at a time so the commit message can link to the change log of the respective dependency.

**Review by sschuberth — CHANGES_REQUESTED** [review]

**Review by sschuberth — APPROVED** [review]

File: gradle.properties (hunk 1 above)
Comment (sschuberth) [reply gradle.properties]: FYI, I decided to remove the *simple-excel* dependency in https://github.com/oss-review-toolkit/ort/pull/3195.
